cmd/gameserver/packet/character.go

```diff
@@ -186,3 +186,26 @@ func SetCharacterSlotOrder(session *network.Session, reader *network.Reader) {
 
 	session.Send(packet)
 }
+
+func NewTargetUser(session *network.Session, reader *network.Reader) {
+	sessionId := reader.ReadUint16()
+
+	pSession := g_NetworkManager.GetSession(sessionId)
+	ctx, err := parseSessionContext(pSession)
```

Comment:
You should rebase these changes on `development` branch, I just merged world manager, mob movement and some other stuff.

`parseSessionContext` is replaced by `contex.Parse(session)`.